vassal-app/src/main/java/VASSAL/counters/Deck.java

```diff
@@ -1756,13 +1756,16 @@ public boolean equals(Object o) {
       Arrays.sort(pieces);
     }
 
+    final JButton okButton;
+    final JButton cancelButton;
+
     final JList<AvailablePiece> list = new JList<>(pieces);
     list.setSelectionMode(ListSelectionModel.MULTIPLE_INTERVAL_SELECTION);
     d.add(new ScrollPane(list));
     d.add(new JLabel(Resources.getString("Deck.select_cards"))); //$NON-NLS-1$
     d.add(new JLabel(Resources.getString("Deck.then_click"))); //$NON-NLS-1$
     final Box box = Box.createHorizontalBox();
-    JButton b = new JButton(Resources.getString(Resources.OK));
+    JButton b = okButton = new JButton(Resources.getString(Resources.OK));
```

Comment:
Could you just get rid of `b` in this function and use the _real_ names of the buttons?

Reply:
I was sort of afraid I might accidentally break something, and this seemed low-risk?